h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -3,6 +3,7 @@
 import hex.*;
 import hex.genmodel.utils.DistributionFamily;
 import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.NotImplementedException;
```

Comment:
I would rather use UnsupportedOperationException. It is not common to use the commons.lang version

Reply:
Its forgotten import.

Removed.